client/my-sites/sharing/connections/connection.jsx

```diff
@@ -78,12 +79,35 @@ class SharingConnection extends Component {
 		};
 	}
 
+	componentWillMount() {
+		this.fetchKeyringUser();
+	}
+
+	componentWillReceiveProps( nextProps, prevProps ) {
+		if ( nextProps.siteId !== prevProps.siteId || nextProps.connection !== prevProps.connection ) {
```

Comment:
cool catch @yurynix! `prevProps` here should be `undefined` according to the React docs…